Lib/asyncio/tasks.py

```diff
@@ -570,62 +570,79 @@ async def _cancel_and_wait(fut):
         fut.remove_done_callback(cb)
 
 
-# This is *not* a @coroutine!  It is just an iterator (yielding Futures).
-def as_completed(fs, *, timeout=None):
-    """Return an iterator whose values are coroutines.
+class as_completed(object):
```

Comment:
```suggestion
class as_completed:
```
Inheriting from object isn't necessary